src/feature-test/user_see_form.test.js

```diff
@@ -26,6 +30,7 @@ describe('I.I.A.S', () => {
     });
     await page.click('input[type="submit"]');
     page.on('response', response => { response.body })
```

Comment:
I don't believe this is actually doing anything as the response is all being handled in the request further up.

Reply:
Resolved